src/components/Like.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useState } from 'react';
+import { patchReview } from '../utils/utils'
+
+
+
+function Like({vote, setVote, review_id, voted, setVoted}) {
+    
+
+    function handleLike () {
+        if(voted){
+            setVoted(false);
+            setVote(vote - 1) 
+        } else {
+            setVoted(true);
+            setVote(vote + 1) 
+            patchReview(review_id, {inc_vote:1})
+           .then((data) => {
+        })
```

Comment:
You could include the .catch() block here instead of the utils file, then please make sure that the user is notified if a patch request faild.